stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/composer/MessageComposerController.kt

```diff
@@ -43,6 +60,32 @@ public class MessageComposerController(
      */
     public val selectedAttachments: MutableStateFlow<List<Attachment>> = MutableStateFlow(emptyList())
 
+    /**
+     * Represents the list of validation errors for the current text input and the currently selected attachments.
+     */
+    public val validationErrors: MutableStateFlow<List<ValidationError>> = MutableStateFlow(emptyList())
+
+    /**
+     * Represents the maximum allowed message length in the message input.
+     */
+    private var maxMessageLength: Int = DEFAULT_MAX_MESSAGE_LENGTH
```

Comment:
Any reason why this is not in the constructor? Cannot access the object constant?

Reply:
Because this value is coming from the backed. Do we need a way to override the backend value?